proto/shentu/oracle/v1alpha1/tx.proto

```diff
@@ -109,3 +113,44 @@ message MsgDeleteTask {
 }
 
 message MsgDeleteTaskResponse {}
+
+message MsgCreateTxTask {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+
+    string creator = 1 [ (gogoproto.moretags) = "yaml:\"creator\"" ];
+    string chain_id = 2 [ (gogoproto.moretags) = "yaml:\"chain_id\"" ];
+    // the bytes of business chain transaction that is going to be evaluated
+    bytes tx_bytes = 3 [ (gogoproto.moretags) = "yaml:\"tx_bytes\"" ];
+    repeated cosmos.base.v1beta1.Coin bounty = 4 [ (gogoproto.moretags) = "yaml:\"bounty\"", (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins" ];
+    google.protobuf.Timestamp valid_time = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"valid_time\""];
+}
+
+message MsgCreateTxTaskResponse {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+    // sha256 hash of the business chain transaction
+    bytes tx_hash = 1;
```

Comment:
taskid?

Reply:
Yeah, `task_id` sounds better than `tx_hash`. 
will rename it and the attribute name of the score event.

Reply:
no I mean taskID and tx hash should both be returned